scenarios/mongoose_pubsub_docker.erl

```diff
@@ -50,7 +50,18 @@ start(MyId) ->
 
 work_as_publisher(MyId, Client) ->
     create_pubsub_node(Client, pubsub_node(MyId)),
```

Comment:
@kklocek I guess the solution is based on the fact that `MyId rem 3 /= 0` always holds true - am I right? If so, then is should be documented and explained in the commit message.

Apart from that, I'm not sure if this scenario is still valid after this change - probably should be checked by @chrzaszcz.

Anyway, I suggest moving this piece of code onto separate branch and creating a new PR for it describing exactly which dialyzer error it addresses. I'm not sure that you picked a right solution.